Package.swift

```diff
@@ -16,6 +16,12 @@ let package = Package(
             name: "DatadogObjc",
             type: .dynamic,
             targets: ["DatadogObjc"]),
+        .library(
+            name: "DatadogStatic",
+            targets: ["Datadog"]),
+        .library(
+            name: "DatadogStaticObjc",
+            targets: ["DatadogObjc"]),
```

Comment:
if Xcode changes its behavior one day, these may end up being linked dynamically
can we please enforce static linking by adding `type: .static` explicitly?

```suggestion
        .library(
            name: "DatadogStatic",
            type: .static,
            targets: ["Datadog"]),
        .library(
            name: "DatadogStaticObjc",
            type: .static,
            targets: ["DatadogObjc"]),
```

Reply:
Very good point. Changes have been made.